.changeset/itchy-eggs-film.md

```diff
@@ -0,0 +1,8 @@
+---
+"@aws-amplify/ui-docs": patch
+"@aws-amplify/ui-angular-example": patch
+"@aws-amplify/ui-angular-mono": patch
+"@aws-amplify/ui-angular": patch
```

Comment:
```suggestion
"@aws-amplify/ui-angular": major
```

Reply:
It would get upgraded to major when `@aws-amplify/ui` is set to `major`, right?

Reply:
yes, but this change itself also deserves a major bump on Angular. This would ensure changelog gets formatted as expected.

Reply:
Changeset will combine the bumps intelligently when it's time to release.

Reply:
I think we want to add a new changeset for each breaking changes PR, since each breaking change PRs affect different packages.